unified-runtime/source/adapters/hip/usm.cpp

```diff
@@ -37,7 +37,13 @@ urUSMHostAlloc(ur_context_handle_t hContext, const ur_usm_desc_t *pUSMDesc,
     return USMHostAllocImpl(ppMem, hContext, /* flags */ 0, size, alignment);
   }
 
-  return umfPoolMallocHelper(hPool, ppMem, size, alignment);
+  auto UMFPool = hPool->HostMemPool.get();
```

Comment:
Should a similar change be done to urUSMSharedAlloc?

Reply:
Good catch, I removed the helper and made the same change for shared alloc too